weave/index/index_pandas.py

```diff
@@ -12,6 +12,27 @@
 from .create_index import create_index_from_fs
 from .index_abc import IndexABC
 
+def slice_df(df, max_rows=None, offset=0):
+    """Returns the pandas dataframe representation of the index.
+
+    Parameters
+    ----------
+    df: pd.DataFrame
+        The dataframe to slice.
+    max_rows: int (default=None)
+        Max rows returned in the pandas dataframe. If None, all rows will be
+        returned.
+    offset: int (default=0)
+        Offset from the beginning of the index to begin the query
+
+    Returns
+    ----------
+    pandas.DataFrame
+        Returns a slice of the dataframe.
+    """
+    return df.iloc[offset:offset+max_rows] if max_rows > 0 else df
```

Comment:
'>' not supported between instances of 'NoneType' and 'int'

Reply:
You're totally right... let me fix that...